theseus/src/api/profile/create.rs

```diff
@@ -102,6 +102,14 @@ pub async fn profile_create(
         }
 
         profile.metadata.linked_data = linked_data;
+        if let Some(linked_data) = &profile.metadata.linked_data {
+            profile.locked = Some(
+                linked_data.project_id.is_some()
+                    && linked_data.version_id.is_some(),
+            );
+        } else {
+            profile.locked = Some(false);
+        }
```

Comment:
i feel like locked should be inside of linked data. doesn't make sense since it will never be relevant if there is no linked data